cypress/support/hostsfunctions.ts

```diff
@@ -77,11 +77,11 @@ function editHost(invenotryName: string, host_type: string, hostName: string, vi
   cy.hasDetail(/^Description$/, 'This is the description edited');
 }
 
-function deleteHostListView(invenotryName: string, host_type: string, hostName: string) {
+function deleteHostListView(inventoryName: string, host_type: string, hostName: string) {
   // function for delete host
   // can use this for stand alon host and for invntory host
   // will delete and verify that all was deleted curectlly
-  navigateToBaseView(host_type, invenotryName);
+  navigateToBaseView(host_type, inventoryName);
   cy.filterTableByMultiSelect('name', [hostName]);
   cy.get(`[data-cy="actions-column-cell"] [data-cy="actions-dropdown"]`).click();
```

Comment:
please change this selector, I assume it is row item kebab. we have many custom commands in place mainly used for CRUD actions (edit, delete from line row item kebab menu)
refer to commands.md file it is very handy

Reply:
this is navigation to host list view for both inventory host test and stand alone host test
how can I find the CRUD action that is relevant for that action?

Reply:
also - can I do that in the nest PR?
really want to move to the next tests that need to be covered without dealing with all the conflicts that come from this PR and other that will be merge in this time